pkg/metrics/register.go

```diff
@@ -45,7 +46,8 @@ func RegisterPrometheusExporter(addr string) error {
 	// Run the Prometheus exporter as a scrape endpoint.
 	go func() {
 		mux := http.NewServeMux()
-		mux.Handle("/metrics", pe)
+		mux.Handle("/metrics", pe)                              // OpenCensus
+		mux.Handle("/experimental-metrics", promhttp.Handler()) // Prometheus Go client
```

Comment:
I grant its experimental right now but not sure that the path we want to use. How about /prom-metrics or /prometheus-metrics ?